test/relay-sproof-builder/src/lib.rs

```diff
@@ -17,12 +17,13 @@
 use cumulus_primitives_core::{
 	relay_chain, AbridgedHostConfiguration, AbridgedHrmpChannel, ParaId,
 };
+use polkadot_primitives::v1::UpgradeGoAhead;
 use sp_runtime::traits::HashFor;
 use sp_state_machine::MemoryDB;
 use sp_std::collections::btree_map::BTreeMap;
 
 /// Builds a sproof (portmanteau of 'spoof' and 'proof') of the relay chain state.
-#[derive(Clone)]
+// #[derive(Clone)]
```

Comment:
This one is related to `TODO`, we cannot derive `Clone` since `UpgradeGoAhead` doesn't implement it.
To resolve it now, it's necessary to update polkadot repo and the PR as well.
It also can be done as a follow-up.

Reply:
Then remove it